tests/ui/associated-consts/assoc-const-eq-missing.stderr

```diff
@@ -2,19 +2,25 @@ error[E0220]: associated type `Z` not found for `Foo`
   --> $DIR/assoc-const-eq-missing.rs:15:16
    |
 LL | fn foo1<F: Foo<Z=3>>() {}
-   |                ^ associated type `Z` not found
+   |                ^
+   |
+   = help: `Foo` has no associated type, try removing `Z`
```

Comment:
I don't think the right suggestion for a user proving an associated type bound that can't be found is to remove it.